packages/tuono-router/src/components/RouteMatch.spec.tsx

```diff
@@ -49,6 +55,11 @@ describe('<RouteMatch />', () => {
   afterEach(cleanup)
 
   it('should correctly render nested routes', () => {
+    vi.mocked(useServerPayloadData).mockReturnValue({
```

Comment:
Since each test now has its own `mockReturnValue`, 
the `vi.mocked(useServerPayloadData)` on line 49:

```ts
vi.mocked(useServerPayloadData).mockReturnValue({
  data: undefined,
  isLoading: false,
})
```

no longer has any effect.

You should replace it with a `useServerPayloadDataMock` variable:

```ts
const useServerPayloadDataMock = vi.mocked(useServerPayloadData)
```

and use it in each test instead of calling `vi.mocked` every time.

Reply:
Agree